src/iconsManifest/supportedExtensions.ts

```diff
@@ -3392,7 +3392,7 @@ export const extensions: IFileCollection = {
     {
       icon: 'nuxt',
       extensions: ['.nuxtignore', '.nuxtrc'],
-      filenamesGlob: ['nuxt.config'],
+      filenamesGlob: ['nuxt.config', '.config/nuxt'],
```

Comment:
Only filenames are supported, not paths.

```suggestion
      filenamesGlob: ['nuxt.config', 'nuxt'],
```